job-transifex-push.yml

```diff
@@ -0,0 +1,41 @@
+# Pushes string files to Transifex.
+
+parameters:
+- name: source_file_pattern
+  type: string
+  default: ".*/res/values/strings"
+- name: tx_token
+  type: string
+- name: transifex_client_version
```

Comment:
Nit: The one Transifex-related parameter is called `tx_token`, the other `transifex_client_version`. Could we unify the names? I.e. for the latter we could use `tx_version` or `tx_cli_version`.

Reply:
Yes, let's go with `transifex_token`? We don't use any shortcuts in other parameter names.

Reply:
Sure!